seed/challenges/02-javascript-algorithms-and-data-structures/object-oriented-programming.json

```diff
@@ -318,7 +318,8 @@
         ""
       ],
       "tests": [
-        "assert(ownProps.includes('name') && ownProps.includes('numLegs'), 'message: <code>ownProps</code> should include the values <code>\"numLegs\"</code> and <code>\"name\"</code>.');"
```

Comment:
"assert(ownProps.includes('panda') && ownProps.includes(' 2legs'), 'message: <code>ownProps</code>black and whit<code>\"numLegs\"</code> and <code>\"panda"</code>.');"
 
"assert (ownPr